clang/docs/checkedc/Update-to-latest-LLVM-sources.md

```diff
@@ -23,13 +23,17 @@ The third step is to merge your changes back into your baseline and master branc
 First create remote to the mirrored GitHub repo that contains the updated sources
 for LLVM/Clang. Go to your LLVM/Clang repo and do:
 
-    git remote add mirror https://github.com/llvm-mirror/llvm-project
+    git remote add mirror https://github.com/llvm/llvm-project
 
 Then branch your baseline branch and merge changes into it:
 
     git checkout baseline
```

Comment:
To be clear, you can delete the `git checkout baseline` command now since the later `git checkout` command switches to another branch before anything is done with `baseline`.